rom/laghos_rom.cpp

```diff
@@ -4388,36 +4799,41 @@ void ROM_Operator::StepRK2Avg(Vector &S, double &t, double &dt) const
         // - Update V using dv_dt.
         // - Compute de_dt and dx_dt using S and V.
 
-        // -- 1.
-        // S is S0.
+		// -- Stage 1 of 2 (S is S_n = S0).
         hydro_oper->UpdateMesh(fx);
```

Comment:
I haven't changed anything in the functionality of the average RK2 timestepper. I just added some comments that help me understand the carried out computations. I can remove them if you find them confusing.